### PR TITLE
README

### PR DESCRIPTION
**Pull request by veredrec**
Add logo, user journey, user stories, team guidelines and titles to readme

### CODE REVIEW

**Review by sajedazoabi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
-# Path18 
-Web App for exposing young people to a variety of career paths :mortar_board:
+##### Web App for exposing young people to a variety of career paths :mortar_board:
+
+### Problem Statement:
+* Unexposure to variety of jobs and career paths for young people
+* Uncertainty around career prospects
+
+### User Journey:
+#### As a user I can browse a variety of career paths and view detailed information about the ones that I am interested in.
```
Comment (matthewdking) [inline README.md]: Personal preference maybe but I don't think this should be '####'

File: README.md [hunk 2]
```diff
@@ -1,3 +1,38 @@
+![Path18 logo](logo.png)
 
-# Path18 
-Web App for exposing young people to a variety of career paths :mortar_board:
+##### Web App for exposing young people to a variety of career paths :mortar_board:
```
Comment (Heathercoraje) [inline README.md]: maybe it is better to say
web app for informing young people of a variety of careers and paths?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,38 @@
...
+* CamelCase for JavaScript id's
+* kebab-case for CSS Classes
+* comments for code clarification
+* ES6 on backend
+* all team member understand code before approval
+
+### Stretch Goals:
+* user authentication
+* use of logic for browsing job options
+* expand data to include colleges
```
Comment (matthewdking) [inline README.md:38]: another stretch goal could be the ability to delete a job once you have been matched with it

**Comment by sajedazoabi** [comment]
:smile: - Thank you Vered

**Comment by matthewdking** [comment]
Maybe start the file structure for the logo instead of having the logo in the root folder
> public/images/logo.png

File: README.md (hunk 3 above)
Comment (veredrec) [reply README.md:38]: I didn't add it yet, but we can definitely add it later :wink:

**Review by Heathercoraje — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**